managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -2572,6 +2572,17 @@ void internalTrimConsumedLedgers(CompletableFuture<?> promise) {
         internalTrimLedgers(false, promise);
     }
 
+    private OffloadPolicies getOffloadPoliciesIfEnabled() {
```

Comment:
It's better to give it a Optional<OffloadPolicies>

Reply:
Changed